CDP4Composition/Mvvm/ViewModelBase.cs

```diff
@@ -315,7 +315,7 @@ protected virtual void Dispose(bool disposing)
             {
                 // Clear all property values that maybe have been set
                 // when the class was instantiated
-                this.RevisionNumber = 0;
+                this.revisionNumber = 0; //Don't set the reactive property this.RevisionNumber here. Otherwise NotifyChange might execute unwanted code.
```

Comment:
@samatrhea any clue why we had to set this in the Dispose in any case?

Reply:
no, no clear idea